recipes/tsam/meta.yaml

```diff
@@ -1,13 +1,13 @@
 {% set name = "tsam" %}
-{% set version = "2.0.0" %}
+{% set version = "2.2.0" %}
```

Comment:
The latest stable tag is https://github.com/FZJ-IEK3-VSA/tsam/releases/tag/v2.0.0
Can you ask upstream to release a 2.2.0? Or maybe base your recipe on the 2.0.0?

Reply:
Thank your very much for your review. There was a mistake in the version number of the commits. The code is based on version 2.0.0. I Incorporated your request.